doc/users-guide/users-guide-utilities-examples.adoc

```diff
@@ -0,0 +1,17 @@
+== Utilities and examples
+
+=== PcapNg capture
+If compiled using `--enable-pcapng-support` ODP will offer packet capturing
+functionality in PcapNg format. If /var/run/odp directory exists prior to
+launching the application ODP will create as many fifos as the NIC queues.
+Queue naming will be of the following format: *<odp global pid>-<NIC
+name>-flow-<queue *number>*
```

Comment:
This formats as __\<odp global pid\>-\<NIC name\>-flow-\<queue *number\>__ 

Is that what you wanted?

Reply:
no the extra * will be removed on v2